src/decoder.c

```diff
@@ -385,3 +385,29 @@ int nanocbor_skip(nanocbor_value_t *it)
 {
     return _skip_limited(it, NANOCBOR_RECURSION_MAX);
 }
+
+bool nanocbor_get_key_tstr(nanocbor_value_t *start, const char *key,
```

Comment:
I would prefer to have this return an `int`, with `NANOCBOR_OK` if the key is found and a new `NANOCBOR_NOT_FOUND` if it is not found. Then an early return can be placed if `nanocbor_get_tstr` and `nanocbor_skip` fail, each returning the error code from that call.

Reply:
Ok, changed it.